### DISCUSSION TITLE
Is it possible to rearrange the queue view to make it more readable on a mobile phone?

### DISCUSSION THREAD

**Discussion by DoDiesis** [jcorporation/myMPD#494] [Ideas]
Is it possible to rearrange the queue view to make it more readable on a mobile phone?
The following images show what I mean.
myMPD currently shows a playlist in the following way:
![photo_2021-05-19_23-17-52](https://user-images.githubusercontent.com/76726999/118885795-7d636c00-b8f8-11eb-97c9-47ca92409752.jpg)
This arrangement looks very ugly on a mobile phone, especially if the songs are classical music tracks, with very long titles.
I think the following arrangement could be more readable:
![photo_2021-05-19_23-20-41](https://user-images.githubusercontent.com/76726999/118886174-e519b700-b8f8-11eb-8635-463c023e9e10.jpg)
I mean the various field should be arranged in lines, instead of columns.

**Comment by DoDiesis** [comment]
I forgot: the small icon near each line isn't needed. I only suggests to arrange the fields by line, instead of columns.

**Comment by jdr85** [comment]
Good question, on my phone it does not look too good either.

Dire Straits :smiling_face_with_three_hearts:

**Comment by jcorporation** [comment]
This could be an option for all table based views. 

My prefered option is an button to toggle between table and grid as in windows explorer.

But more opinions would be good!

**Reply by DoDiesis** [reply]
What about an option in the settings menu?

**Reply by jcorporation** [reply]
For the default? Rows, grid or auto 
Row - actual view
Grid - as you suggested 
Auto - Grid for small screens

**Reply by DoDiesis** [reply]
I think "Auto" should be preferable.
